workers/unity/Packages/com.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -32,17 +32,46 @@ public static void Build()
             try
             {
                 var args = CommandLineArgs.FromCommandLine();
-                var buildTargetArg = args.GetCommandLineValue("buildTarget", "local");
+                var buildEnvironmentArg = args.GetCommandLineValue("buildEnvironment", "local");
+                var buildTargetArg = args.GetCommandLineValue("buildTarget", "All");
```

Comment:
Also why not support multiple build targets with comma delineated arguments given we are supporting this. I.e. - `win,linux`

Reply:
wish granted 🎉